core/src/main/java/org/mapfish/print/servlet/job/impl/hibernate/HibernateJobQueue.java

```diff
@@ -209,11 +214,26 @@ public final void shutdown() {
 
   private void cleanup() {
     TransactionTemplate tmpl = new TransactionTemplate(this.txManager);
-    final int nbDeleted =
-        tmpl.execute(
-            status ->
-                HibernateJobQueue.this.dao.deleteOld(
-                    System.currentTimeMillis() - getTimeToKeepAfterAccessInMillis()));
-    metricRegistry.counter(getClass().getName() + ".deleted_old").inc(nbDeleted);
+    String name = MetricRegistry.name(getClass().getSimpleName(), "deletedOldPrintJobs");
+    try (Timer.Context deletion = metricRegistry.timer(name).time()) {
+      final Integer nbDeleted =
+          tmpl.execute(
+              status -> {
+                try {
+                  return HibernateJobQueue.this.dao.deleteOld(
+                      System.currentTimeMillis() - getTimeToKeepAfterAccessInMillis());
+                } catch (Exception e) {
+                  unhealthyCountersHealthCheck.recordUnhealthyProblem(
+                      name, "failedToDeleteOldJobs");
```

Comment:
DeleteOldJobsError

Reply:
HibernateJobQueue.deletedOldPrintJobs.DeleteOldJobsError is also redundant.
How about 
HibernateJobQueue.deletedOldPrintJobs.issue or
HibernateJobQueue.deletedOldPrintJobs.problem ?

Reply:
+1 for HibernateJobQueue.deletedOldPrintJobs.issue :-)